pkgs/applications/networking/p2p/transgui/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchsvn, pkgconfig, makeDesktopItem, unzip, fpc, lazarus,
+libX11, glib, gtk, gdk_pixbuf, pango, atk, cairo }:
+
+stdenv.mkDerivation rec {
+  name = "transgui-trunk-r${version}";
+  version = "986";
```

Comment:
Installing with "nix-env -f . -iA transgui" worked... Can you suggest an alternative name? I want to distinguish between the releases (5.0.1) and the svc trunk (986) since the daemon could change the RPC and break protocol compatibility between releases.